tuyaha/tuyaapi.py

```diff
@@ -14,6 +14,7 @@
 REFRESHTIME = 60 * 60 * 12
 
 _LOGGER = logging.getLogger(__name__)
+REQUEST_SESSION = requests.Session()
```

Comment:
Please, move `REQUEST_SESSION` inside TuyaAPI.init method, like this:
```
self.requestSession = requests.Session()
```

do not pay attention to global `SESSION` variable. It's an old code from previous developers that needs refactoring.

Reply:
Fixed on ef4c333